src/desktop/GDBTargetDebugSession.ts

```diff
@@ -221,6 +225,11 @@ export class GDBTargetDebugSession extends GDBDebugSession {
 
         this.launchGdbServer = true;
         this.killGdbServer = target.automaticallyKillServer !== false;
+        this.serverDisconnectTimeout =
+            target.serverDisconnectTimeout !== undefined &&
+            target.serverDisconnectTimeout > 0
```

Comment:
```suggestion
            target.serverDisconnectTimeout >= 0
```
We need to support value `0`.